modules/cr_email_signup/src/DataIngestion/ESUDataIngestion.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+namespace Drupal\cr_email_signup\DataIngestion;
+
+
+/**
+ * Email Sign up Data Ingestion class.
+ */
+class ESUDataIngestion
+{
+
+    protected $data_ingestion_endpoint = 'https://ingest-staging.data.comicrelief.com/identify';
```

Comment:
Might be better setting this as a Drupal config setting in `cr_email_signup_settings.yml`, that way we can set-up different endpoints for dev and prod within our CR/SR projects

Reply:
Sounds good